hpx/hpx_fwd.hpp

```diff
@@ -1703,6 +1654,9 @@ namespace hpx
 #endif
 }
 
+// Including declarations of get_locality_name
+#include <hpx/runtime/get_locality_name.hpp>
+
```

Comment:
Why is this a separate header (I think we should keep all of the API in `hpx_fwd.hpp`)? Also, this header is currently missing from the repository?